### PR TITLE
chore: enable single contact id

### PR DESCRIPTION
**Pull request by tapiarafael**
## Proposed changes (including videos or screenshots)
Enable single contact id in production, allowing to use without the `TEST_MODE` flag

### CODE REVIEW

File: .changeset/eight-coins-agree.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Added omnichannel contact feature
```
Comment (KevLehman) [inline .changeset/eight-coins-agree.md:5]: Changesets are in present tense now, pls update 🙏🏽

File: apps/meteor/app/statistics/server/lib/getAppsStatistics.ts
Comment (KevLehman) [inline apps/meteor/app/statistics/server/lib/getAppsStatistics.ts:1]: probably a remanent of a bad merge/rebase? 👀

File: apps/meteor/app/statistics/server/lib/statistics.ts
Comment (KevLehman) [inline apps/meteor/app/statistics/server/lib/statistics.ts:1]: Same

File: apps/meteor/server/models/raw/LivechatRooms.ts
Comment (KevLehman) [inline apps/meteor/server/models/raw/LivechatRooms.ts:1]: Same

**Review by KevLehman — COMMENTED** [review]
PR looks good just remvoe the files from the stats changes and it should be good to be merged.

**Review by pierre-lehnen-rc — APPROVED** [review]

**Pull request merged**